### PR TITLE
[Keyboard] Add via compatibility to Daisy 40 [1/2]

### PR DESCRIPTION
**Pull request by jesdga95**
## Description

Adds compatibility for VIA on Daisy 40, for both split and full size spacebar

fullsize
split

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## VIA Pull Request

https://github.com/the-via/keyboards/pull/226

## Issues Fixed or Closed by This PR

* N/A

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/daisy/config.h [hunk 1]
```diff
@@ -20,12 +20,12 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
...
 /* USB Device descriptor parameter */
-#define VENDOR_ID       0xFEED
-#define PRODUCT_ID      0x7169
+#define VENDOR_ID       0x4B50
+#define PRODUCT_ID      0xD7DC
 #define DEVICE_VER      0x0501
 #define MANUFACTURER    KTEC
 #define PRODUCT         Daisy
-#define DESCRIPTION     qmk port for Daisy
+#define DESCRIPTION     QMK + VIA port for Daisy
```
Comment (zvecr) [inline keyboards/daisy/config.h:28]:
```suggestion
```

File: keyboards/daisy/rules.mk [hunk 2]
```diff
@@ -30,3 +30,5 @@ BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
 AUDIO_ENABLE = no           # Audio output on port C6
 FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
 HD44780_ENABLE = no         # Enable support for HD44780 based LCDs
+VIA_ENABLE = yes
+BOOTMAGIC_ENABLE = lite
```
Comment (zvecr) [inline keyboards/daisy/rules.mk:34]: This shouldnt be enabled here but in a `via` keymap, <https://caniusevia.com/docs/configuring_qmk>
```suggestion
```

File: keyboards/daisy/config.h (hunk 1 above)
Comment (jesdga95) [reply keyboards/daisy/config.h:28]: Not sure I understand? Revert this change?

File: keyboards/daisy/rules.mk (hunk 2 above)
Comment (jesdga95) [reply keyboards/daisy/rules.mk:34]: Will do

File: keyboards/daisy/config.h (hunk 1 above)
Comment (zvecr) [reply keyboards/daisy/config.h:28]: I wouldn't need to contain the notion of via, but that doesnt matter since #9732 and the decision to remove.

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**